pkg/pull/pull.go

```diff
@@ -557,6 +557,21 @@ func Pull(upstreamURI string, pullOptions PullOptions) (string, error) {
 		}
 	}
 
+	installationBytes, err := ioutil.ReadFile(filepath.Join(u.GetUpstreamDir(writeUpstreamOptions), "userdata", "installation.yaml"))
+	if err != nil {
+		return "", errors.Wrap(err, "failed to read installation file")
+	}
+
+	// Ensure the installation manifest filename is unique.
```

Comment:
no need. vendors can't/won't create installation manifests. let's just use `installation.yaml` like we do in the `upstream` directory.